vtm/src/org/oscim/tiling/source/mapfile/OSMUtils.java

```diff
@@ -43,6 +43,31 @@
      * Determining what is an area is neigh impossible in OSM, this method inspects tag elements
      * to give a likely answer. See http://wiki.openstreetmap.org/wiki/The_Future_of_Areas and
      * http://wiki.openstreetmap.org/wiki/Way
+     * <p>
+     * The order in which the if-clauses are checked is determined with the help from
+     * https://taginfo.openstreetmap.org/ - last accessed 2020-12-23
+     * <p>
+     * key statistics:
+     * 429 422 565 building
+     * 188 142 457 highway
+     * 44 369 957 natural
+     * 31 354 174 landuse
+     * 17 740 976 amenity
+     * 14 581 922 barrier
+     * 6 366 976 leisure
+     * 5 361 889 railway
+     * 1 467 189 area
+     * 550 597 aeroway
+     * <p>
+     * railway statistics:
+     * 2 138 877 rail
+     * 93 926 tram
+     * 60 396 subway
+     * 45 182 narrow_gauge
+     * 30 668 light_rail
+     * 24 203 construction
+     * 8 937 preserved
+     * 3 138 monorail
```

Comment:
We do not need so detailed statistics in the code.
An info that the order is based on OSM metrics is sufficient.

Reply:
i'm okay with removing these lines and adding the OSM metric info, shall i change it or can you do it?

Reply:
The order's source is already mentioned above, so we can just remove the numbers.